### PR TITLE
Make `normalize()` method to static in `CacheKeyNormalizer::class`.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

### CODE REVIEW

**Review by vjik — CHANGES_REQUESTED** [review]
Need line in changelog and info in UPGRADE.md

**Review by samdark — APPROVED** [review]

**Comment by terabytesoftw** [comment]
> Need line in changelog and info in UPGRADE.md

done.

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 ## 3.0.1 under development
 
 - New #132: Add interface `SerializerInterface` for data serialization, and `PhpSerializer` implementation (@Gerych1984)
+- Enh #139: Make `normalize()` method to static in `CacheKeyNormalizer::class` (@terabytesoftw)
```
Comment (vjik) [inline CHANGELOG.md:6]:
```suggestion
- Chg #139: Make `normalize()` method static in `CacheKeyNormalizer` class (@terabytesoftw)
```

File: UPGRADE.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Upgrading Instructions for Yii Cache
+
+* Change method `normalize()` to static in `CacheKeyNormalizer` class.
```
Comment (vjik) [inline UPGRADE.md:3]:
```suggestion
# Upgrading Instructions for Yii Cache

This file contains the upgrade notes. These notes highlight changes that could break your
application when you upgrade the package from one version to another.

> **Important!** The following upgrading instructions are cumulative. That is, if you want
> to upgrade from version A to version C and there is version B between A and C, you need
> to following the instructions for both A and B.

## Upgrade from 3.x

- Change method `normalize()` to static in `CacheKeyNormalizer` class.
```

**Review by vjik — APPROVED** [review]

**Pull request merged**